code/obj/item/clothing/hats.dm

```diff
@@ -1303,3 +1303,33 @@
 	wear_image_icon = 'icons/mob/fruithat.dmi'
 	icon_state = "headsprout"
 	item_state = "headsprout"
+
+
+/obj/item/clothing/head/hos_hat
+	name = "HoS Hat"
+	icon_state = "hoscap"
+	uses_multiple_icon_states = 1
+	item_state = "hoscap"
+	c_flags = SPACEWEAR
+	var/is_a_communist = 0
+	var/folds = 0
+	desc = "Actually, this hat is from a fast-food restaurant, that's why it folds like it was made of paper."
+	setupProperties()
+		..()
+		setProperty("meleeprot_head", 3)
```

Comment:
All other changes are fine, but the lowering of this value, without this change, it's fine to merge.

Reply:
You know what, I'm fine with that. This is not a hill I want to die on. Compromise and yadda yadda.